### ISSUE TITLE
📚 Documentation: images in cards should have empty alt attributes [a11y][TheA11y100]

### ISSUE THREAD

**Issue by GrahamTheDevRel** [appwrite/website#284]
## Issue
In the "show me some code" section, the cards are links. Within them are images with `alt` attributes, however these `alt` attributes are not descriptive and, **more importantly**, add duplication of information making the link text longer than needed.

![Card on Docs homepage has alt text highlighted](https://github.com/appwrite/website/assets/121672086/ddb97808-8fb1-4b3e-8edc-795c7f1d6525)

## Why it matters
People using assistive technology will end up with extra, repeated or irrelevant information as part of the link text.

## Suggested fix
Remove the text within the `alt` attribute. **do not remove the attribute**. 

An empty `alt` attribute hides the image from assistive technology so it does not add unnecessary noise within the link. Removing the `alt` attribute would result in the images URL being read out which would make the problem worse!

Also, it is suggested that you add `role="presentation"` to the image. This does not doing anything, **but** it does make it easier for people in the future to realise that the image `alt` attribute has *deliberately* been left empty and is a good practice. @Haimantika could you confirm that Appwrite are OK with this policy please.

## Relevant Success Criterion WCAG
[Link Purpose (In Context)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)

## Additional Notes
Please wait for confirmation from Appwrite on whether to add `role="presentation"` to images that are decorative (deliberately empty `alt` attributes mentioned earlier.)

[Found on stream on 2023-10-25](https://www.twitch.tv/videos/1959992199)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by Sweetdevil144** [comment]
Hey @GrahamTheDevRel . Is the still still open?
If so, I would like to work on it!! 🙂

**Comment by Susmita-Dey** [comment]
Hey, I'd like to take up this issue.

**Comment by TGlide** [comment]
@Sweetdevil144 assigned 😄
